### PR TITLE
fix: 🐛 allow reject a proposal by creator with no other votes + modify required signatures

### PR DESCRIPTION
**Pull request by sansan**
### Description

- allows a multiSig propoposal to be rejected by creator if no other votes have been casted
- allows modifying the required signatures for a multiSig

### JIRA Link

https://polymesh.atlassian.net/browse/DA-1288

### CODE REVIEW

File: src/api/procedures/__tests__/modifyMultiSig.ts [hunk 1]
```diff
@@ -330,7 +384,135 @@ describe('modifyMultiSig procedure', () => {
...
+        code: ErrorCode.NoDataChange,
+        message:
+          'The given signers are equal to the current signers. At least one signer should be added or removed',
       });
+
+      try {
+        assertNoDataChange(new BigNumber(2), [], [], [oldSigner1, oldSigner2]);
+      } catch (error) {
+        expect(error).toEqual(expectedError);
+      }
```
Comment (polymath-eric) [inline src/api/procedures/__tests__/modifyMultiSig.ts:479]:
```suggestion
        expect(() => assertNoDataChange(new BigNumber(2), [], [], [oldSigner1, oldSigner2])).toThrow(expectedError);
```

I think passing a function is cleaner syntax vs try catch.

Comment (polymath-eric) [reply src/api/procedures/__tests__/modifyMultiSig.ts:479]: With catch syntax would the test fail if the assert function didn't throw?

It looks like the expect would be skipped and the test would still pass, so its not just a cosmetic change.

I think with try catch you need to assign to an error variable and assert outside the catch.

**Review by polymath-eric — DISMISSED** [review]

**Review by prashantasdeveloper — APPROVED** [review]

**Pull request merged**